### PR TITLE
Return average prediction for segments with SAF > 0.45

### PR DESCRIPTION
**Pull request by acatellierNTIA**
This PR brings the C++ version to feature parity with the MATLAB version.

The C++ version will now return all predictions for all segments in brackets as well as an average of the predictions on speech segments with a speech activity factor > 0.45.

There are also cleanup work in the documentation and a windows batch file that parallels `demo.m`.

### CODE REVIEW

File: cplusplus/Wawenet/helpScreen.txt [hunk 1]
```diff
@@ -28,14 +30,18 @@ L=1: normalization on (Default)
...
-speech file will produce two results.
-If S = 24,000 for example, segment overlap will be 50%, a 4.5 sec.
-file will produce 2 results and a 6 sec. file will produce 3 results.
+If the input speech .wav file is longer than 3 seconds
+multiple results may be produced. S specifies the number of samples to
+move ahead in the file when extracting the next segment.
+The default value of 48,000 gives zero overlap between
+segments. Using this default any input less than 6 sec. will produce one
+result, based on just the first 3 sec. A 6 sec. input will produce two
+results. If S = 24,000 for example, segment overlap will be 50%, a
```
Comment (RBallardNTIA) [inline cplusplus/Wawenet/helpScreen.txt:39]: Probably ultimately a Lilli question, but my two cents:

Using the shortened version of seconds `sec.` here seems a bit ambiguous, particularly with the other case of `3 sec.` which has the `.` actually function as a full stop for the sentence as opposed to indicating the shortened version of seconds. Maybe it'd be better to use `sec` or `seconds` for this text block, then the `.` isn't ambiguous?

**Review by RBallardNTIA — APPROVED** [review]
Looks good to me. Just one minor grammar comment on some documentation I noticed, but still approving.

**Comment by svoranNTIA** [comment]
also please fix commented out lines in WAWEnet.m so that filenames are returned, and fix demo.m to reflect that this is now the case

**Review by svoranNTIA — APPROVED** [review]
Thank you Andrew.  The Matlab changes we discussed look fine, and you even found a few bonus changes!

**Pull request merged**